src/solvers/Makefile

```diff
@@ -217,7 +217,7 @@ INCLUDES += -I .. \
 
 CLEANFILES = solvers$(LIBEXT) smt2_solver$(EXEEXT)
 
-all: solvers$(LIBEXT) smt2_solver$(EXEEXT)
+all: solvers$(LIBEXT) # smt2_solver$(EXEEXT)
```

Comment:
The commit message does not say "why" it was disabled, and in particular this does not explain why earlier commits first enabled/added it while now it is being disabled.

Reply:
reworded